File: VB/Dashboard_CustomVisualInteractivity_Web/Dashboard1.vb
Imports System
Imports Dashboard_CustomVisualInteractivity_Web.nwindDataSetTableAdapters

Namespace Dashboard_CustomVisualInteractivity_Web
    Partial Public Class Dashboard1
        Inherits DevExpress.DashboardCommon.Dashboard

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Dashboard1_DashboardLoading(ByVal sender As Object, ByVal e As EventArgs) _
            Handles MyBase.DashboardLoading
            Dim categories As New SalesPersonTableAdapter()
            categories.Fill(Me.nwindDataSet1.SalesPerson)
        End Sub
    End Class
End Namespace

File: CS/Dashboard_CustomVisualInteractivity_Web/Dashboard1.cs
using System;
using Dashboard_CustomVisualInteractivity_Web.nwindDataSetTableAdapters;

namespace Dashboard_CustomVisualInteractivity_Web {
    public partial class Dashboard1 : DevExpress.DashboardCommon.Dashboard {
        public Dashboard1() {
            InitializeComponent();
        }

        private void Dashboard1_DashboardLoading(object sender, EventArgs e) {
            SalesPersonTableAdapter categories = new SalesPersonTableAdapter();
            categories.Fill(this.nwindDataSet1.SalesPerson);
        }
    }
}
